README.md

```diff
@@ -32,12 +32,17 @@ Check the [releases](https://github.com/minicraftplus/minicraft-plus-revived/rel
 ## Current goals and ideas
 Take a look at the [ideas](ideas/) folder or the [issues](https://github.com/minicraftplus/minicraft-plus-revived/issues) page.
 
-## How to build/run
+## Getting the game and run the game
+Head over [releases](https://github.com/minicraftplus/minicraft-plus-revived/releases) and find the latest version of Minicraft+.
+There, you can find an asset called similarly `minicraft_plus.jar` and click the file. After you get the file downloaded, you can double-click the file in downloading folder.
+You must have first confirmed that you have Java (at least 8) installed on your computer.
```

Comment:
`You must first confirm that you have [Java](https://www.java.com/en/download/) (at least version 8) installed on your computer.`